### DISCUSSION TITLE
Please help me name a movie!🥳🥳🥳🥳🥳🥳🥳🥳

### DISCUSSION THREAD

**Discussion by 278Mt** [NAIST-SE/AttractingContributors#49] [General]
url is: https://github.com/278Mt/AttractingContributors

**Comment by satokibi44** [comment]
added!
Cloud you contribute my repo?
https://github.com/satokibi44/AttractingContributors-Anime

**Reply by 278Mt** [reply]
thx;-)
I added.

**Comment by eedy8888** [comment]
added please help me
https://github.com/eedy8888/AttractingContributors

**Reply by 278Mt** [reply]
thx;-)
I add my favorite Japanese food, too<3

**Comment by nosfill** [comment]
added!

**Reply by 278Mt** [reply]
thx;-)

**Comment by wowry** [comment]
Added, and would you please help me?
https://github.com/wowry/AttractingContributors-VideoGames

**Reply by 278Mt** [reply]
thx;-)

**Comment by tktkbohshi** [comment]
I added!
Could you help me?
https://github.com/tktkbohshi/AttractingContributors-favorite-niconico-douga

**Reply by 278Mt** [reply]
thx;-)

**Comment by sergiodl-naist** [comment]
added one of my favorites
Please visit and try to ask a question or recommend one vtuber on my fork!
https://github.com/sergiodl-naist/AttractingContributors-VTubers
